api/http/handler/hostmanagement/intel/handler.go

```diff
@@ -31,6 +35,7 @@ func NewHandler(bouncer *security.RequestBouncer, dataStore portainer.DataStore)
 	featureEnabled, _ := settings.FeatureFlagSettings[portainer.FeatOpenAMT]
 	if featureEnabled {
 		h.Handle("/open-amt", bouncer.AdminAccess(httperror.LoggerHandler(h.openAMTConfigureDefault))).Methods(http.MethodPost)
+		h.Handle("/open-amt/{id}/info", bouncer.AdminAccess(httperror.LoggerHandler(h.OpenAMTHostInfo))).Methods(http.MethodGet)
```

Comment:
What about spliting this action into two separated endpoints, one for pulling and deploying the docker image (POST endpoint) and another one for fetching the host information (execute `amtinfo` on created container). Would that be possible?

Reply:
after I asked thisI noticed that the whole idea is to remove the container after runningit so I guess my idea doesn't really apply :)

Reply:
to be honest, we need to cache the result, not the container/image so we don't request it constantly - but this initial POC code is trying more to create the handler to exec a container abstraction, not the datamodel part.

it'll need to get better in later PR's tho